tests/test_aps.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of hepcrawl.
+# Copyright (C) 2015 CERN.
+#
+# hepcrawl is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import pytest
+
+from hepcrawl.spiders import aps_spider
+from .responses import fake_response_from_file
+
+
+@pytest.fixture
+def results():
+    """Return results generator from the WSP spider."""
+    from scrapy.http import TextResponse
+
+    spider = aps_spider.APSSpider()
+    return spider.parse(fake_response_from_file('aps/sample_aps_record.json', response_type=TextResponse))
+
+
+def test_abstract(results):
+    """Test extracting abstract."""
+    abstract = (
+        "We use a popular fictional disease, zombies, in order to introduce techniques used in modern epidemiology"
+        " modeling, and ideas and techniques used in the numerical study of critical phenomena. We consider variants of"
+        " zombie models, from fully connected continuous time dynamics to a full scale exact stochastic dynamic"
+        " simulation of a zombie outbreak on the continental United States. Along the way, we offer a closed form"
+        " analytical expression for the fully connected differential equation, and demonstrate that the single person"
+        " per site two dimensional square lattice version of zombies lies in the percolation universality class. We end"
+        " with a quantitative study of the full scale US outbreak, including the average susceptibility of different"
+        " geographical regions."
+    )
+    record = results.next()
```

Comment:
Some spider `parse` functions do not yield a generator always (but a list), and to avoid different techniques between tests, I propose to just put it in a `for` loop like this:

```python
for record in results:
    # test code
```